CONTRIBUTING_MODELS.md

```diff
@@ -0,0 +1,235 @@
+# Model Contribution Guide
+
+KerasNLP has a plethora of cutting-edge "backbone" model ranging from BERT and
+RoBERTa to FNet and DeBERTa. We are always looking for more models and are always
+open to contributions!
+
+In this guide, we will walk you through the steps one needs to take in order to contribute
+a backbone model. For illustration purposes, let's assume that you want to
+contribute the DistilBERT model. Before we dive in, we encourage you to go through
+[our getting started guide](https://keras.io/guides/keras_nlp/getting_started/)
+for an introduction to the library, and our
+[contribution guide](https://github.com/keras-team/keras-nlp/blob/master/CONTRIBUTING.md).
+
+## Checklist
+
+This to-do list is a brief outline of how a backbone model can be contributed.
+Keep this checklist handy!
+
+### Step 1: Open an issue/find an issue
+
+- [ ] Open an issue or find an issue to contribute a backbone model.
+
+### Step 2: PR #1 - Add XXBackbone
+
+- [ ] An `xx/xx_backbone.py` file which has the model graph [[Example Link](https://github.com/keras-team/keras-nlp/blob/master/keras_nlp/models/distil_bert/distil_bert_backbone.py)].
```

Comment:
nit maybe `Example Link` -> `Example` (the link is already in blue, calling it link is a little redudant)